app/assets/javascripts/discourse/app/components/user-menu/notifications-list.js

```diff
@@ -84,9 +88,23 @@ export default class UserMenuNotificationsList extends UserMenuItemsList {
     const content = [];
     const data = await ajax("/notifications", { data: params });
 
-    const notifications = await Notification.initializeNotifications(
+    let notifications = await Notification.initializeNotifications(
       data.notifications
     );
+    // mark likes as read on all notifications list
```

Comment:
I don't think we need to do this, we never did, having them unread there feels fine